cfep-01.md

```diff
@@ -0,0 +1,135 @@
+
+<table>
+<tr><td> Title </td><td> CFEP Purpose and Guidelines </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Jonathan J. Helmus &lt;jjhelmus@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Aug 30, 2016</td></tr>
+<tr><td> Updated </td><td> Aug 30, 2016</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+# Abstract
+
+CFEPs are the manner in which the conda-forge community suggests changes to the
+project, and outlines how the suggested changes are reviewed.  This document provides details on what a CFEP is, how to author one, and the workflow used to review these proposals.
+
+# What is a CFEP
+
+CFEP stands for Conda-Forge Enhancement Proposal.  A CFEP is a document which
+outlines a suggested change to how the project operates.  These changes may be
+technical in manner or may deal with social aspects of the project including
+governance and conduct.  The CFEP should provide a concise description of the
+proposed change as well as the rationale for the change.
+
+CFEPs are intended to be the primary mechanism for proposing major changes to
+how conda-forge operates and for documenting both the changes themselves and the
+decision making process.
+
+CFEPs are maintained as text files using Markdown for formatting in the
+[conda-forge/conda-forge-enhancement-proposals](https://github.com/conda-forge/conda-forge-enhancement-proposals)
+repository. Their revision history is a historic record of the proposed change.
+
+# Workflow
+
+The CFEP process begins with an idea for a change in how the conda-forge
+organization operates.  These change can be technical or address the
+social aspects of the organization.  Small changes often do not need a CFEP and
+can be discussed on the
+[conda-forge gitter channel](https://gitter.im/conda-forge/conda-forge.github.io),
+the [conda-forge mailing list](https://groups.google.com/forum/#!forum/conda-forge),
+or as part of a conda-forge meeting.  More involved or controversial changes
+should be submitted as CFEPs.  When it is unclear if a change requires a CFEP,
+ask in one of the above forums and the conda-forge core team should be able to
```

Comment:
I think the mailing list is a better forum for this kind of question because it is easier to find the discussion later. gitter is pretty stream-of-conciousness.